src/haz3lcore/statics/Info.re

```diff
@@ -64,6 +64,7 @@ type error_common =
 [@deriving (show({with_path: false}), sexp, yojson)]
 type error_exp =
   | FreeVariable(Var.t) /* Unbound variable (not in typing context) */
+  | InexhaustiveMatch(option(error_exp))
```

Comment:
`Self.exp.InexhaustiveMatch` gets translated to `Info.error_exp.InexhaustiveMatch`. There might be an additional error associated with the match expression. Currently, it seems that it can only be an inconsistent-branch error (`Common(Inconsistent(Internal(_)))`).